test/unit/module/helpers/test_get_url_content.py

```diff
@@ -11,6 +11,9 @@
 from test.testlib.testcase import BaseTestCase
 from mock import patch, MagicMock
 import cfnlint.helpers
+import random
+import string
```

Comment:
These can both be removed if people were fine with hard-coding the ETag value in the tests. I just used them to generate a random string.